tests/utils_tests/test_autoreload.py

```diff
@@ -674,8 +674,7 @@ class BaseReloaderTests(ReloaderTests):
 
     def test_watch_dir_with_unresolvable_path(self):
         path = Path("unresolvable_directory")
-        with mock.patch.object(Path, "absolute", side_effect=FileNotFoundError):
-            self.reloader.watch_dir(path, "**/*.mo")
+        self.reloader.watch_dir(path, "**/*.mo")
```

Comment:
Minor but I wonder if we should `with assertLogs("django.utils.autoreload", level="DEBUG")` as well to assert against the emitted messages.

Reply:
Good catch, will adjust